app/utils/UserAnswers.scala

```diff
@@ -182,6 +182,15 @@ case class UserAnswers(json: JsValue = Json.obj()) extends Enumerable.Implicits{
         details.companyName, details.isDeleted, isComplete.getOrElse(false), isNew.fold(false)(identity), noOfRecords)
     )
 
+    private def readsCompanyHnS(index: Int): Reads[Establisher[_]] = (
```

Comment:
This method doesn't seem to be any different from the preceding one, i.e. readsCompany therefore why do we need it?